### ISSUE TITLE
Cached first photo on IOS9 with ionic

### ISSUE THREAD

**Issue by paszc** [JrSchild/jr-crop#33]
Taking and cropping first photo works great but when i take a second photo, in jr-crop there is still the first photo
On Android everything works great. It's only IOS issue, i tried everything clearing variables, cache. I can take another (one) photo only after closing app.

My code :

```
navigator.camera.getPicture(onCapturePhoto, onFail, {
                quality: 60,
                encodingType: Camera.EncodingType.JPEG,
                sourceType:Camera.PictureSourceType.CAMERA,
                targetWidth: 1000,
                targetHeight: 1000,
                correctOrientation: true,
                allowEdit: false,
                destinationType: destinationType.FILE_URI
            });

```

```
function onCapturePhoto(fileURI) {
            var win = function (r) {
                clearCache();
                retries = 0;
                var response = JSON.parse(r.response);
                sharedProperties.setObject('remoteFileURI', response.path);
                sharedProperties.setObject('isSend', true);
            }

          var fail = function (error) {
                if (retries == 0) {
                      alert( 'retrie!');
                    retries++;
                    setTimeout(function () {
                        onCapturePhoto(fileURI)
                    }, 1000);
                } else {
                    retries = 0;
                    clearCache();
                     alert('Ups. Something wrong happens!');
                }
            }

            sharedProperties.setObject('localFileURI', fileURI);

            $jrCrop.crop({
                url: fileURI,
                width: window.innerWidth - (window.innerWidth * 0.1),
                height: window.innerWidth - (window.innerWidth * 0.1)
            }).then(function (canvas) {
                // success!
                sharedProperties.setObject('croppedImage', canvas.toDataURL());
                clearCache();
                $state.go('nextstep');

            }, function () {
                $state.transitionTo('index');
            });

```

**Comment by JrSchild** [comment]
I am aware of the situation. What you can do for now is cache bust the fileURI by adding the following:

```
+ "?t=" + Date.now()
```

Duplicate of #29
